tools/common.c

```diff
@@ -158,6 +166,51 @@ struct poptOption connect_options[] = {
 #endif /* WITH_SSL */
     {NULL, '\0', 0, NULL, 0, NULL, NULL}};
 
+void read_authfile(const char *path)
+{
+  size_t n;
+  FILE *fp;
+  char token[MAXAUTHTOKENLEN];
+
+  if ((amqp_username = malloc(MAXAUTHTOKENLEN)) == NULL ||
+      (amqp_password = malloc(MAXAUTHTOKENLEN)) == NULL) {
+    die("Out of memory");
+  } else if ((fp = fopen(path, "r")) == NULL) {
+    die("Could not read auth data file", path);
```

Comment:
die() expects a format string, plus printf style arguments e.g., die("Could not read auth data file: %s", paht)

Reply:
Done.